### PR TITLE
Add steps to store smokeping rrd in librenms folder

### PR DESCRIPTION
**Pull request by AnaelMobilia**
Add steps to store smokeping rrd in librenms folder

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Rosiak** [comment]
Why would you do that?

**Comment by AnaelMobilia** [comment]
> Why would you do that?

Hello,
The goal of this PR is to provide detailled steps of the operation writed on [the smokeping documentation](https://docs.librenms.org/Extensions/Smokeping/) on the rrdcached part :
> **You will need to either change the dir in which smokeping saves its rrd files to be the same as the main librenms dir** or you can remove the -B argument from the rrdcached config to allow it to read from more than one dir.

Regards,
Anael

File: doc/Extensions/Smokeping.md [hunk 1]
```diff
@@ -304,3 +304,39 @@ BASE_OPTIONS=
...
+Then update the config file:
+
+```bash
+datadir = /opt/librenms/rrd/smokeping
+dyndir = /opt/librenms/rrd/smokeping/__cgi
+```
+And give to smokeping rights to access files
+
+```bash
+sudo adduser smokeping librenms
```
Comment (murrant) [inline doc/Extensions/Smokeping.md]: This isn't how you add a user to a group...

Comment (AnaelMobilia) [reply doc/Extensions/Smokeping.md]: @murrant I'm not sure of what you mean.
[adduser allow this use at least on Debian](https://manpages.debian.org/buster/adduser/adduser.8.en.html#Add_an_existing_user_to_an_existing_group).

Regards,
Anael

Comment (murrant) [reply doc/Extensions/Smokeping.md]: Heh, I don't even have the adduser command on my distro.

It is probably safer to use:
usermod -a -G librenms smokeping

Comment (AnaelMobilia) [reply doc/Extensions/Smokeping.md]: Updated, thanks !

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-57-release-changelog-october-2019/9969/1

**Pull request merged**